src/main/java/seedu/address/logic/parser/PictureCommandParser.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_FILE_LOCATION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import java.util.stream.Stream;
+
+import seedu.address.logic.commands.PictureCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.Name;
+
+/**
+ * Parses input arguments and creates a new PictureCommand object
+ */
+public class PictureCommandParser implements Parser<PictureCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the PictureCommand
+     * and returns an PictureCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    @Override
+    public PictureCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimap =
+            ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_FILE_LOCATION);
+
```

Comment:
Please follow the form of EditCommand's parse and use the Indices instead. The reason you should let the person who is doing "edit by name" handle this is change to "edit by name" is because theres' a lot more unnecessary logic that needs to be handled that isn't implemented in this version, e.g. name clashes.